manifests/server.pp

```diff
@@ -44,6 +44,7 @@
   Boolean $manage_config                                                        = $prometheus::manage_config,
   Optional[Variant[Stdlib::HTTPurl, Stdlib::Unixpath, String[1]]] $external_url = $prometheus::external_url,
   Optional[Array[Hash[String[1], Any]]] $collect_scrape_jobs                    = $prometheus::collect_scrape_jobs,
+  Optional[Integer] $max_open_files                                             = undef,
```

Comment:
Can we default to `$prometheus::max_open_files` this would drop the need of using `pick` in `config.pp` if I'm not mistaken.

Reply:
I defaulted as suggested to `$prometheus::max_open_file` and get it in config.pp from the server class. Did you mean it like this?

Reply:
Yes, thank you! :)